designer/server/src/main/scala/pl/touk/nussknacker/ui/api/ProcessesResources.scala

```diff
@@ -206,6 +206,21 @@ class ProcessesResources(
               }
             }
           }
+        } ~ path("migrateScenario") {
```

Comment:
This could be just `createScenario`. If we could change `path("processes" / Segment / Segment)` API definition to just `path("processes")` with entity `CreateProcessCommand` we could remove this endpoint at all.

Reply:
The name is wrong: this resource doesn't handle the whole migration process. We still do an update (`saveProcess`) on the triggering designer side in case if the scenario is missing.
Also I think that we need to have only one resource for scenario creation. WDYT about passing this user in header in the original endpoint? How bad it would be? :)